pages/common/sbt.md

```diff
@@ -11,6 +11,10 @@
 
 `sbt new {{scala/hello-world.g8}}`
 
+- Delete all generated files (in the target directory):
```

Comment:
Do you mean working (current) directory?
```suggestion
- Delete all generated files (in the current directory):
```

Reply:
@blueskyson No, I mean `target` directory, which is a directory that specific to scala builds. It is for cleaning previously compiled "stuff" prior to a new compile.

Reply:
I'd still remove the parentheses (I can't make a suggestion for some reason)

Reply:
@marchersimon Made this change based on your suggestion. Ready for re-review :)